testlib/src/main/resources/scala/scalafmt/scalafmt.conf

```diff
@@ -1,2 +1,4 @@
+version = 3.5.9
+runner.dialect = scala213
```

Comment:
The latest version of scalafmt 3.5.9 has these as mandatory fields. If these fields are not specified then `ScalaExtensionTest` fails.